README_WINDOWS.md

```diff
@@ -276,6 +245,7 @@ Required components:
 - **[CMake][cmake]** >= 3.7.2. You will need at least this version to generate project files for
   VS2017. 3.5 is the absolute supported minimum.
 - **[Visual Studio 14 2017][VS2017]**. 2015 is also supported.
+- **Visual C++**. In VS, this may be obtained by checking the "Desktop development with C++" workload during installation. If you already installed VS without checking this, open up the Visual Studio Installer app, and click "Modify" on your VS installation to get to the workloads listing.
```

Comment:
I don't think this note is necessary; this is a rare problem and it's easy to find the fix with a google search. If you really feel it's needed I'd recommend making it less wordy by merging it with the line about requiring Visual Studio. I am really averse to making this readme any more verbose than it already is. There's a point where it stops being helpful and starts becoming an accumulated mass of every developer's minor foibles, and at that point nobody wants to actually follow it closely.

Reply:
Also, if you do edit the line above can you change "14 2017" to "15 2017"? My personal foible right there :)

Reply:
if bloat is an issue, dont worry, i've got another branch where ive been cleaning up this README and reducing clutter :)))